src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,153 @@
+
+            try {
+                if (userInput.toLowerCase().equals("list")) {
+                    drawLines();
+                    System.out.println("Here are the tasks in your list:");
+                    for (int j = 0; j < listIndex; j++) {
+                        System.out.println((j + 1) + "." + tasks[j].toString());
+                    }
+                    drawLines();
+                } else if (userInput.toLowerCase().startsWith("done")) {
+                    drawLines();
+                    int taskDone = Integer.parseInt(commandGiven[1]);
+                    tasks[taskDone - 1].markAsDone();
+                    System.out.println("Nice! I've marked this task as done:");
+                    System.out.println(tasks[taskDone - 1].toString());
+                    drawLines();
+                } else if (userInput.toLowerCase().startsWith("todo")) {
+                    drawLines();
+                    System.out.println("Got it. I've added this task:");
+                    userInput = userInput.substring(4).trim();
+                    tasks[listIndex] = new ToDo(userInput);
+                    System.out.println("\t" + tasks[listIndex++].toString());
+                    printListIndex(listIndex);
+                    drawLines();
+                } else if (userInput.toLowerCase().startsWith("deadline")) {
+                    userInput = userInput.substring(8).trim();
+                    int byIndex = userInput.indexOf('/');
+                    String dateInput = userInput.substring(byIndex + 3).trim();
+                    userInput = userInput.substring(0, byIndex - 1);
+                    tasks[listIndex] = new Deadline(userInput, dateInput);
+                    System.out.println("\t" + tasks[listIndex++].toString());
+                    drawLines();
+                    System.out.println("Got it. I've added this task:");
+                    printListIndex(listIndex);
+                    drawLines();
+                } else if (userInput.toLowerCase().startsWith("event")) {
+                    userInput = userInput.substring(5).trim();
+                    int byIndex = userInput.indexOf('/');
+                    String dateInput = userInput.substring(byIndex + 3).trim();
+                    userInput = userInput.substring(0, byIndex - 1);
+                    tasks[listIndex] = new Event(userInput, dateInput);
+                    System.out.println("\t" + tasks[listIndex++].toString());
+                    drawLines();
+                    System.out.println("Got it. I've added this task:");
+                    printListIndex(listIndex);
+                    drawLines();
+                } else if (userInput.toLowerCase().equals("bye")) {
+                    byeMessage();
+                    break;
+                } else if (userInput.toLowerCase().equals("help")) {
+                    commandsAvailable();
+                } else if (userInput.isEmpty()) {
```

Comment:
I don't think this else if condition will be executed with just a todo, deadline or event. Since it is not empty. Consider using a trim and then checking if it's still empty and then this statement will be executed :)